codegen/protocol-test-codegen-local/Tests/swift-codegen/Tests/WaitersTests/WaiterTestUtils.swift

```diff
@@ -11,7 +11,7 @@ import SmithyWaitersAPI
 
 // Convenience test-helper methods for testing acceptor matches
 
-extension WaiterConfiguration.Acceptor.Match: Equatable where Input: Equatable, Output: Equatable {
+extension WaiterConfiguration.Acceptor.Match: @retroactive Equatable where Input: Equatable, Output: Equatable {
```

Comment:
Suppress a couple of Swift 6 warnings for extending types from outside the module, by adding `@retroactive` to the conformance.